site_root/input/resources/terminology/ValueSet-phenotype-codes.json

```diff
@@ -13,6 +13,12 @@
     "include": [
```

Comment:
Don't we include SNOMED CT as well?

Reply:
@liberaliscomputing Are snomed just used for  "phenotype-observation-codes" ?

Reply:
Ah yes. We don't need to include SNOMED CT here.